plugins/rollup-plugin-add-browser-to-output.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
In shopify app pixel, browser contains all the methods for local storage, session storage and cookies. Hence we need to pass browser as a global variable to our bundled code